go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/aws/aws-app-mesh-controller-for-k8s
 
+go 1.13
```

Comment:
Was this required for net/http/pprof support?

Reply:
No it's not required but pinning the go version in go.mod is good practice in my opinion. There were many problems with go 1.12 and modules, upgrading to 1.13 is a safe bet :)

Reply:
Got it. Thanks! :)